tests/python/test_bit_operations.py

```diff
@@ -13,3 +13,13 @@ def shl(a: ti.i32, b: ti.i32) -> ti.i32:
 
     for i in range(8):
         assert shl(3, i) == 3 * 2**i
+
+
+@ti.test()
+def test_bit_sar():
+    @ti.kernel
+    def sar(a: ti.i32, b: ti.i32) -> ti.i32:
+        return a >> b
+
+    for i in range(8):
+        assert sar(2**8, i) == 2**(8 - i)
```

Comment:
Please test negative numbers here as well to ensure SAR keeps the sign bit unchanged.

Reply:
Thanks for your advice, I will add test case for negative numbers